### PR TITLE
10 error component

### PR DESCRIPTION
**Pull request by dousdogukan**
### Description

implement CErrorCard

### Checklist

- [x] Tested changes locally.
- [x] Ensured code follows project standards.
- [ ] Updated documentation.
- [ ] Added necessary tests.

### Additional Notes

for #10

### CODE REVIEW

**Comment by ceydae** [comment]
@ruveyda-kisla What's your thoughts about this?

File: src/assets/Close.tsx
Comment (ruveyda-kisla) [inline src/assets/Close.tsx:1]: import all Icon components you created to `index.tsx` in `src/assets`

**Review by ruveyda-kisla — CHANGES_REQUESTED** [review]

File: src/components/CErrorCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import ErrorIcon from '@/assets/ErrorIcon';
```
Comment (ceydae) [inline src/components/CErrorCard/index.tsx:1]: move below the package imports

**Review by ceydae — CHANGES_REQUESTED** [review]

**Review by ceydae — APPROVED** [review]

File: src/components/CPageCard/index.tsx [hunk 2]
```diff
@@ -14,9 +16,9 @@ interface CPageCardProps {
   childrenClassName?: string;
   borderStatus: 'bordered' | 'borderless';
   dividerResponsiveClassName?: string;
+  borderStatus?: 'bordered' | 'borderless';
   children: JSX.Element | React.ReactNode;
-  errorTitle?: string;
-  errorMessage?: string;
+  error?: ErrorType | null;
```
Comment (ceydae) [inline src/components/CPageCard/index.tsx:21]: remove `| null`

**Review by ceydae — CHANGES_REQUESTED** [review]

File: src/components/CPageCard/index.tsx [hunk 3]
```diff
@@ -41,12 +61,23 @@ const CPageCard = ({
...
+          </div>
+        )}
       </div>
 
-      <div className={`${cn(childrenClassName, `h-full overflow-y-auto mobile:px-2 px-4`)}`}>
+      <div
+        className={`${cn(childrenClassName, `h-full overflow-y-auto mobile:px-2 px-4`, {
+          'pointer-events-none opacity-50 select-none': errorVisible,
+        })}`}
+      >
```
Comment (ruveyda-kisla) [inline src/components/CPageCard/index.tsx:80]: This causes CModal to not work as it should

**Pull request merged**